metadata.md

```diff
@@ -52,26 +53,28 @@ Results for a measurement of a single property of SARS-CoV-2 wastewater test, fo
     - `covidE`: SARS-CoV-2 gene region E
     - `covidRdRp`: SARS-CoV-2 gene region RdRp
     - `covidN1N2avg`: SARS-CoV-2 gene region average of N1 and N2
-    -   `covidN1N2avg`: 
-    -   `wwParamFlow`: 
-    -   `wwParamTss`: 
-    -   `wwParamBod`: 
-    -   `catOther`: 
+    -  `covidN1N2avg`: 
+    -  `wwParamFlow`:  Flow rate of the waste water at point of sampling
```

Comment:
Thanks for putting these back in. I will note and update for PR https://github.com/Big-Life-Lab/covid-19-wastewater/pull/39